Sources/XCGrapherLib/Helpers/ShellTask.swift

```diff
@@ -33,6 +33,7 @@ extension ShellTask {
         if task.terminationStatus == 0 {
             return String(data: output, encoding: .utf8)!
         } else if task.terminationStatus == 127 {
+            // FIXME: There are cases that it returns status 127 and it's not because the command was not found, e.g. syntax issues with the shell command
```

Comment:
Both, I'd say. Since we're not yet validating for valid --scheme and --target values passed in, it's likely that those would cause problems with the Ruby scripts. Idk if those problems would trigger this specific error 127, though, I don't remember 😬 Maybe not (maybe just literally invalid syntax commands, which would be just during development)